src/selenium/generatorPage.js

```diff
@@ -0,0 +1,114 @@
+/* eslint-disable no-empty-label */
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Rule 'space-after-keywords' was removed and replaced by: keyword-spacing (space-after-keywords)](https://app.codacy.com/app/triplez/open-event-webapp/pullRequest?prid=3226522)